Resources/Prototypes/Entities/Objects/Devices/Circuitboards/computer.yml

```diff
@@ -325,6 +325,10 @@
     prototype: ComputerRadar
   - type: ComputerTabletopBoard # Frontier
     prototype: ComputerTabletopRadar # Frontier
+  - type: Tag # Frontier
+    tags: # Frontier
+    - DroneUsable # Frontier
+    - RadarConsoleCircuitboard # Frontier
```

Comment:
In the case when there are already tags from upstream and possibly more tags coming later,
we should add empty components as to avoid conflicts, let me know if you need me to so this for you and ill swap it to work the same just comp based.

Reply:
I have no idea what this means

Reply:
Same as here:
https://github.com/new-frontiers-14/frontier-station-14/pull/1487

Reply:
That doesn't help me. I don't know what I'm supposed to be looking for there.

Reply:
Darin, I need you to actually explain it to me. I don't have your experience so I don't know what difference this makes to what I'm PR'ing.

Reply:
Alright, yeah that makes sense. Cheers for explaining. 
Are you happy to do it for me? I'm not quite confident enough but I'll look at what you've done so I can switch to doing it that way in the future.

Reply:
Sure give me some time to fix it, in the main time this PR is still good to go per review, the swap will be under the hood change.